scripts/makecrates.py

```diff
@@ -28,7 +28,7 @@
     "stm32f4": ["atomics", "critical-section", "defmt", "rt", "stm32f401", "stm32f407", "stm32f413", "stm32f469"],
     "stm32f7": ["atomics", "critical-section", "defmt", "rt", "stm32f733", "stm32f779"],
     "stm32h5": ["atomics", "critical-section", "defmt", "rt", "stm32h503", "stm32h523", "stm32h562", "stm32h573"],
```

Comment:
Completely unrelated to this PR but I believe the "stm32h563" feature was removed in #1124 https://github.com/stm32-rs/stm32-rs/pull/1124/commits/6cf40035f18ce99355903ef3672a0fd5b103566b#r1924396832

Reply:
those are not "features" which isbacceptable. those are features which will be rendered on docs.rs

Reply:
Okay, thanks